src/components/timecard/select-timecard-period-type/SelectTimecardPeriodType.jsx

```diff
@@ -1,15 +1,19 @@
 import PropTypes from 'prop-types';
 import { useForm } from 'react-hook-form';
 import { useEffect } from 'react';
-import { useTimecardContext } from '../../../context/TimecardContext';
-
 import Radios from '../../common/form/radios/Radios';
-import { useApplicationContext } from '../../../context/ApplicationContext';
 import { ContextTimeEntry } from '../../../utils/time-entry-utils/ContextTimeEntry';
 import { addTimePeriodHeading } from '../../../utils/time-entry-utils/timeEntryUtils';
 import { focusErrors } from '../../../utils/common-utils/common-utils';
 
-const SelectTimecardPeriodType = () => {
+const SelectTimecardPeriodType = ({
```

Comment:
I can't see how a control that allows you to simply select a timecard period type has this many dependencies

Reply:
To be looked at in future work